tutorial/20_recipes/004_cli.py

```diff
@@ -17,7 +18,7 @@
     studies, Show a list of studies.
     study optimize, Start optimization of a study.
     study set-user-attr, Set a user attribute to a study.
-    tell, Finish a trial, which was created by the ask command.
+    tell, Finish a trial\\, which was created by the ask command.
```

Comment:
Since `,` is used to create the table, I've added `\\` as an escape for the comma